arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -560,9 +560,343 @@
+          "externalDocs": {
+              "url": "https://msdn.microsoft.com/en-us/library/azure/mt639380.aspx"
+          },
+        "parameters": [
+          {
+              "name": "resourceGroupName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the resource group."
+          },
+          {
+              "name": "namespaceName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The namespace name."
+          },
+          {
+              "name":"queueName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The queue name."
+          },
+          {
+              "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+            "200": {
+                "description": "Successfully retrieved queue description.",
+                "schema": {
+                    "$ref": "#/definitions/QueueResource"
+                }
+            }
+        }
+      }
     }
   },
   "definitions": {
+     "QueueCreateOrUpdateParameters": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets or sets Queue name."
+        },
+        "type": {
+          "type": "string",
+          "description": "Gets or sets ARM type of the namespace."
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets  the location of the resource."
+        },       
+        "properties": {
+          "$ref": "#/definitions/QueueProperties",
+          "description": "Gets or sets properties of the Queues."
+        }
+      },
+      "required": [
+        "location"        
+      ],
+      "description": "Parameters supplied to the CreateOrUpdate Queue operation."
+    },
+    "QueueProperties": {
+      "properties": {
+        "AccessedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Get last time a message was sent, or the last time there was a receive request to this queue."
+        },        
+        "AutoDeleteOnIdle": {
+           "type": "string",
+           "format": "time-span",
+           "description": "Gets or sets the TimeSpan idle interval after which the queue is automatically deleted. The minimum duration is 5 minutes."
+          },
+         "AvailabilityStatus ": {
+          "type": "string",
+          "description": "Gets the entity availability status for the queue.",
+          "enum": [
+            "Available",
+            "Limited",
+            "Renaming",
+            "Restoring",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceType",
```

Comment:
I don't see a correlation between `NamespaceType and AvaialiabilityStatus`. Is this the correct name for the enum?